iceoryx_hoofs/CMakeLists.txt

```
@@ -26,94 +26,106 @@ find_package(iceoryx_platform REQUIRED)
 include(IceoryxPackageHelper)
 include(IceoryxPlatformSettings)
 
-include(cmake/IceoryxHoofsDeployment.cmake)
-
 include("${CMAKE_CURRENT_LIST_DIR}/cmake/IceoryxPlatform.cmake")
 
+include(cmake/IceoryxHoofsDeployment.cmake)
+
 if(CMAKE_SYSTEM_NAME MATCHES Linux OR CMAKE_SYSTEM_NAME MATCHES Darwin)
     option(BUILD_SHARED_LIBS "Create shared libraries by default" ON)
 endif()
 
 set(PREFIX iceoryx/v${CMAKE_PROJECT_VERSION})
 
 #
-########## build iceoryx hoofs lib ##########
+# ######### build iceoryx hoofs lib ##########
 #
 iox_add_library(
-    TARGET                      iceoryx_hoofs
-    NAMESPACE                   iceoryx_hoofs
-    PROJECT_PREFIX              ${PREFIX}
-    PUBLIC_LIBS                 iceoryx_platform::iceoryx_platform
-    PRIVATE_LIBS                ${ICEORYX_SANITIZER_FLAGS}
-    PRIVATE_LIBS_LINUX          acl atomic ${CODE_COVERAGE_LIBS}
-    BUILD_INTERFACE             ${PROJECT_SOURCE_DIR}/include
-                                ${PROJECT_SOURCE_DIR}/legacy/include
-                                ${PROJECT_SOURCE_DIR}/memory/include
-                                ${PROJECT_SOURCE_DIR}/container/include
-                                ${PROJECT_SOURCE_DIR}/vocabulary/include
-                                ${PROJECT_SOURCE_DIR}/utility/include
-                                ${PROJECT_SOURCE_DIR}/primitives/include
-                                ${PROJECT_SOURCE_DIR}/design/include
-                                ${CMAKE_BINARY_DIR}/generated/iceoryx_hoofs/include
-    INSTALL_INTERFACE           include/${PREFIX}
-    EXPORT_INCLUDE_DIRS         include/
-                                legacy/include/
-                                memory/include/
-                                container/include/
-                                vocabulary/include/
-                                utility/include/
-                                primitives/include/
-                                design/include/
+    TARGET
+    iceoryx_hoofs
+    NAMESPACE
+    iceoryx_hoofs
+    PROJECT_PREFIX
+    ${PREFIX}
+    PUBLIC_LIBS
+    iceoryx_platform::iceoryx_platform
+    PRIVATE_LIBS
+    ${ICEORYX_SANITIZER_FLAGS}
+    PRIVATE_LIBS_LINUX
+    acl
+    atomic
+    ${CODE_COVERAGE_LIBS}
+    BUILD_INTERFACE
+    ${PROJECT_SOURCE_DIR}/include
+    ${PROJECT_SOURCE_DIR}/legacy/include
+    ${PROJECT_SOURCE_DIR}/memory/include
+    ${PROJECT_SOURCE_DIR}/container/include
+    ${PROJECT_SOURCE_DIR}/vocabulary/include
+    ${PROJECT_SOURCE_DIR}/utility/include
+    ${PROJECT_SOURCE_DIR}/primitives/include
+    ${PROJECT_SOURCE_DIR}/design/include
+    ${CMAKE_BINARY_DIR}/generated/iceoryx_hoofs/include
```

Can we have an additional layer additional indentation?

```suggestion
    BUILD_INTERFACE
        ${PROJECT_SOURCE_DIR}/include
        ....
    INSTALL_INTERFACE
        ...
```

---

I tried my best to maintain the same form of indentation, but it seems that I do not have a control on the number of spaces/tabs between the columns. Please check the last commit